torch/csrc/jit/codegen/cuda/scheduler/pointwise.cpp

```diff
@@ -850,7 +850,8 @@ void schedulePointwise(Fusion* fusion, const PointwiseParams& params) {
     }
   }
 
-  TransformPropagator(reference_tv).run();
+  TransformPropagator propagator(reference_tv);
```

Comment:
Nit: Since this is the only variant used at the moment, should we maintain an alias for it or just wait until we see what variants are actually going to be used?

Reply:
For now, this is only two lines of code, so I think it is OK to leave it like this for now. If in the future, we need to repeatedly write some boilerplate code, for example, define some subclass of `Selector`, etc., we could consider adding an alias for them.

I will merge this PR as is to unblock my computeAt work but feel free to follow up and I can resolve your suggestions in later PR.